bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/AudioManager.java

```diff
@@ -42,28 +47,69 @@
+     *
+     * @param audioStream The audio stream to play
+     * @param sinkId The id of the audio sink to use or null
+     * @param volume The volume to be used or null if the default notification volume should be used
+     */
+    void play(AudioStream audioStream, @Nullable String sinkId, @Nullable PercentType volume);
 
     /**
      * Plays an audio file from the "sounds" folder using the default audio sink.
      *
+     * @param fileName The file from the "sounds" folder
      * @throws AudioException in case the file does not exist or cannot be opened
      */
     void playFile(String fileName) throws AudioException;
 
+    /**
+     * Plays an audio file with the given volume from the "sounds" folder using the default audio sink.
+     *
+     * @param fileName The file from the "sounds" folder
+     * @param volume The volume to be used or null if the default notification volume should be used
+     * @throws AudioException in case the file does not exist or cannot be opened
+     */
+    void playFile(String fileName, @Nullable PercentType volume) throws AudioException;
+
     /**
      * Plays an audio file from the "sounds" folder using the given audio sink.
      *
+     * @param fileName The file from the "sounds" folder
+     * @param sink The id of the audio sink to use or null
+     * @throws AudioException in case the file does not exist or cannot be opened
+     */
+    void playFile(String fileName, @Nullable String sink) throws AudioException;
+
+    /**
+     * Plays an audio file with the given volume from the "sounds" folder using the given audio sink.
+     *
+     * @param fileName The file from the "sounds" folder
+     * @param sink The id of the audio sink to use or null
+     * @param volume The volume to be used or null if the default notification volume should be used
      * @throws AudioException in case the file does not exist or cannot be opened
      */
-    void playFile(String fileName, String sink) throws AudioException;
+    void playFile(String fileName, @Nullable String sink, @Nullable PercentType volume) throws AudioException;
 
     /**
      * Stream audio from the passed url using the default audio sink.
      *
+     * @param url The url to stream from or null if streaming should be stopped
+     * @throws AudioException in case the url stream cannot be opened
+     */
+    void stream(@Nullable String url) throws AudioException;
+
+    /**
+     * Stream audio with the given volume from the passed url using the default audio sink.
+     *
+     * @param url The url to stream from or null if streaming should be stopped
+     * @param volume The volume to be used or null if the default notification volume should be used
      * @throws AudioException in case the url stream cannot be opened
      */
-    void stream(String url) throws AudioException;
+    void stream(@Nullable String url, @Nullable PercentType volume) throws AudioException;
```

Comment:
I'm actually wondering if we should offer that for streams at all - a stream should never be a notification (but rather some background audio like radio) and it does not have an end when an old volume would be restored.
I'd therefore feel better to always play streams with the current volume and leave it to the user to adjust it at any time.